### ISSUE TITLE
allFlags returns json strings instead of js objects

### ISSUE THREAD

**Issue by rhysawilliams2010** [launchdarkly/react-native-client-sdk#15]
**Describe the bug**
I am using the android emulator to run a react-native application.
When I request a json variation by calling client.allFlags I get back a json string. iOS correctly returns a js object. 
If I request the variation directly (`client.jsonVariation("my-flag")`) I get a js object.

**To reproduce**
Steps to reproduce the behavior.
1. Create a json variation.
1. Run the react-native application on android emulator (Pixel 2 API 29).
1. Request all flags. 

**Expected behavior**
I would expect the allFlags response in android to contain my variation as a js object. E.g:
```
{
  "my-key" : {"hello":"world"}
}
```
Instead it returns the json variation as a string.
```
{
  "my-key" : "{\"hello\":\"world\"}"
}
```

**SDK version**
2.0.0

**OS/platform**
Android SDK 28

**Comment by torchhound** [comment]
Hi @rhysawilliams2010 thanks for reporting this issue. This is a result of Android SDK backward compatibility. We are internally evaluating how to resolve this issue.

**Comment by torchhound** [comment]
Hi again @rhysawilliams2010, this bug has been fixed in the 2.0.1 patch release!
